### ISSUE TITLE
iPhone Landscape Orientation Content Accessability

### ISSUE THREAD

**Issue by maxrugen** [maxrugen/website#1]
The iPhone's (read: any phone's) landscape orientation moves the content upwards, making it almost unreadable.

Suggested fix:
Usage of orientation : landscape media query.
```css
/* iPhone 6 & 7 */
@media only screen 
and (min-device-width : 375px) 
and (max-device-width : 667px) 
and (orientation : landscape)
```

**Comment by maxrugen** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix iPhone Landscape Mode](https://trello.com/c/mDCNQOtd/1-fix-iphone-landscape-mode)

**Comment by maxrugen** [comment]
Fixed with #4 [THE FANCY EDITION](https://github.com/maxrugen/website/pull/4).
